Formula/aurora-cli.rb

```diff
@@ -11,9 +11,15 @@ class AuroraCli < Formula
     sha256 "62e565f0d65601f88b11d8c5f5bfad89d2f1661144072876fc705e50b4284329" => :yosemite
   end
 
+  patch do
```

Comment:
This should be inside a `if MacOS.version == :sierra ... end`

Reply:
@ilovezfs Thanks